models/nft/nft_transfers.sql

```diff
@@ -0,0 +1,124 @@
+{{ config(
+        alias ='transfers',
+        partition_by='block_date',
+        materialized='incremental',
+        file_format = 'delta',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "sector",
+                                    "nft",
+                                    \'["hildobby"]\') }}',
+        unique_key = ['unique_transfer_id']
+)
+}}
+
+SELECT 'ethereum' AS blockchain
+, block_time
+, block_date
+, block_number
+, token_standard
+, transfer_type
+, evt_index
+, contract_address
+, token_id
+, amount
+, from
+, to
+, tx_hash
+, unique_transfer_id
+FROM {{ ref('nft_ethereum_transfers') }}
+UNION
+SELECT 'bnb' AS blockchain
+, block_time
+, block_date
+, block_number
+, token_standard
+, transfer_type
+, evt_index
+, contract_address
+, token_id
+, amount
+, from
+, to
+, tx_hash
+, unique_transfer_id
+FROM {{ ref('nft_bnb_transfers') }}
+UNION
+SELECT 'avalanche' AS blockchain
```

Comment:
I am not sure if we need to use 'avalanche_c' for the blockchain(as we did in the prices.usd table).
```suggestion
SELECT 'avalanche_c' AS blockchain
```